modules/ref-dashboard-configuration-options.adoc

```diff
@@ -11,41 +11,57 @@ The {productname-short} dashboard includes a set of core features enabled by def
 | `dashboardConfig:
-disableUserManagement` | `false` | Shows the *Settings → User management* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+disableHome` | `false` | Shows the *Home* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
-disableProjects` | `false` | Shows the *Data Science Projects* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
- | `dashboardConfig:
- disablePipelines` | `false` | Shows the *Data Science Pipelines* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+disableInfo` | `false` | On the *Applications → Explore* page, when a user clicks on an application tile, an information panel opens with more details about the application. To disable the information panel for all applications on the *Applications → Explore* page , set the value to `true`.
 | `dashboardConfig:
-disableModelServing` | `false` | Shows the *Model Serving* option in the dashboard navigation menu and in the list of components for the data science projects. To hide *Model Serving* from the dashboard navigation menu and from the list of components for data science projects, set the value to `true`. 
+disableISVBadges` | `false` | Shows the label on a tile that indicates whether the application is “{org-name} managed”, “Partner managed”, or “Self-managed”. To hide these labels, set the value to `true`. 
 | `dashboardConfig:
-disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
+disableKServe` | `false` | Enables the ability to select KServe as a Serving Platform. To disable this ability, set the value to `true`.
 | `dashboardConfig:
-disableCustomServingRuntimes` | `false` | Shows the *Serving runtimes* option in the dashboard navigation menu. To hide this menu option, set the value to `true`. 
+disableKServeAuth` | `false` | Enables the ability to use auth with KServe. To disable this ability, set the value to `true`.
 | `dashboardConfig:
-disableKServe` | `false` | Enables the ability to select KServe as a Serving Platform. To disable this ability, set the value to `true`.
+disableKServeMetrics` | `false` | Enables the ability to view KServe metrics. To disable this ability, set the value to `true`.
 | `dashboardConfig:
 disableModelMesh` | `false` | Enables the ability to select ModelMesh as a Serving Platform. To disable this ability, set the value to `true`.
 | `dashboardConfig:
-disableAcceleratorProfiles` | `false`| Shows the *Accelerator profiles* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+disableModelRegistry` | `false` | Shows the *Model Registry* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
```

Comment:
The `disableModelRegistry` flag will also show/hide the Model Registry Settings option in the nav, maybe we should mention both here @bredamc ?

Reply:
Updated - thanks @mturley !